test/command_line/test_import.py

```diff
@@ -474,6 +471,38 @@ def test_import_still_sequence_as_experiments_subset(dials_data, tmpdir):
     assert all(exp.goniometer is not None for exp in imported_exp)
 
 
+def test_import_still_sequence_as_expts_subset_by_range(dials_data, tmp_path):
+    image_files = dials_data("centroid_test_data").listdir("centroid*.cbf", sort=True)
+
+    out = tmp_path / "experiments_as_still.expt"
+
+    result = procrunner.run(
+        [
+            "dials.import",
+            "scan.oscillation=10,0",
+            "image_range=3,5",
+            "output.experiments=%s" % out,
+        ]
+        + [f.strpath for f in image_files],
```

Comment:
```suggestion
            f"output.experiments={out}",
            *image_files,
        ],
```
I find it really hard to notice that the comprehension is part of the build. I checked, and procrunner/subprocess appears to deal fine with both `py.path` and `pathlib` objects.